src/main/java/seedu/address/model/task/DueDateIsBeforeEndOfMonthPredicate.java

```diff
@@ -0,0 +1,32 @@
+package seedu.address.model.task;
+
+import java.util.Calendar;
+import java.util.GregorianCalendar;
+import java.util.function.Predicate;
+
+/**
+ * Tests that a {@code Task}'s {@code DueDate} is before the end of today.
+ */
+public class DueDateIsBeforeEndOfMonthPredicate implements Predicate<Task> {
+
+    @Override
+    public boolean test(Task person) {
+        /** Get time at the end of the month, Sunday 2359. */
```

Comment:
I'm not too sure about this, but i think /**blahblah*/ is for javadoc comment? I think inside a method, it's better to use //blahlah